modules/300-prometheus/hooks/metrics_storage_retention.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hooks
+
+import (
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook/metrics"
+	"github.com/flant/addon-operator/sdk"
+)
+
+var _ = sdk.RegisterFunc(&go_hook.HookConfig{
+	Queue:        "/modules/prometheus/metrics_storage_retention",
+	OnBeforeHelm: &go_hook.OrderedConfig{Order: 10},
+}, storageRetentionMetricHandler)
+
+func storageRetentionMetricHandler(input *go_hook.HookInput) error {
+	retentionDaysMain := input.Values.Get("prometheus.retentionDays")
+	retentionDaysLongterm := input.Values.Get("prometheus.longtermRetentionDays")
+
+	input.MetricsCollector.Set(
```

Comment:
Let it be a single metric `d8_prometheus_storage_retention_days` with labels prometheus="main|longrerm"

Reply:
Fixed